src/overlays/actors/ovl_Bg_Mizu_Water/z_bg_mizu_water.c

```diff
@@ -21,19 +43,305 @@ const ActorInit Bg_Mizu_Water_InitVars = {
     (ActorFunc)BgMizuWater_Update,
     (ActorFunc)BgMizuWater_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mizu_Water/func_8089F8B0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mizu_Water/func_8089F9D0.s")
+static u32 unk_808A0550 = 0;
+static f32 unk_808A0554 = 110.0f;
```

Comment:
Yes, they are unused. Maybe they are from earlier stages of development since 110.0f is the also the y-difference used in type 3 (just speculation though).

Reply:
I don't think we should necessarily name unused things like this. @Roman971 @fig02 ?